version1/04 - Web services.md

```diff
@@ -988,8 +988,8 @@ Content-type: application/json
   "fieldId": string,
   "fieldName": string,
   "fieldDescription": string,
-  "fieldType": string|long|double|timestamp,
-  "fieldValues": [string|long|double|timestamp]
+  "fieldType": string|long|double|timestamp|encrypted,
+  "fieldValues": [string|long|double|timestamp|encrypted]
```

Comment:
Same, users can't create pre-defined encrypted values, so this line should be reverted.

Reply:
Fixed in be13583bd7c237b563a43db275fa59dfdc1f2385.